Data/Sys/GameSettings/GZ2E01.ini

```diff
@@ -362,7 +362,7 @@ $Can Walk Ontop of Deep Snow (Twilight Spectre)
 $Run Normal Speed With Iron Boots On (brkirch)
 040BB53C 38600000
 040B2D18 4800004C
-$Alywas Preform Dive When Jumping (brkirch)
+$Alywas Perform Dive When Jumping (brkirch)
```

Comment:
there's a second typo

Reply:
Also the 'Alywas', hah.  I'll take care of them

Reply:
730f573